src/request/derive-address/DeriveAddress.css

```diff
@@ -1,31 +1,66 @@
-.passphrase-icon-text {
-    display: flex;
-    flex-direction: row;
+.page#unlock {
+    position: relative;
 }
 
-.passphrase-icon-text p:first-of-type {
-    margin-top: 0;
+.page#unlock .nq-text {
+    opacity: 1;
+    font-weight: 600;
+    text-align: center;
 }
 
-.passphrase-icon-text p:last-of-type {
-    margin-bottom: 0;
+.page#unlock .lock-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-grow: 1;
 }
 
-.nq-icon.walletfile-locked {
-    width: 9rem;
-    height: 11.75rem;
-    margin-right: 4rem;
+.nq-icon.locked {
+    width: 8rem;
+    height: 11.52875rem;
     flex-shrink: 0;
+    background-image: url('data:image/svg+xml,<svg width="66" height="93" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M60 45V26C60 12 48 0 33 0S6 12 6 26v19c-9 13-6 30 6 40s30 10 42 0 15-27 6-40zM33 69c-4 0-8-3-8-8 0-4 4-7 8-7s8 3 8 7c0 5-4 8-8 8zm16-35l1-2v-6c0-9-8-16-17-16s-17 7-17 16v6l1 2h2c9-4 19-4 28 0h2z" fill="%231F2348"/></svg>');
+    opacity: 0.4;
 }
 
-.page#choose-identicon .page-body p:last-of-type {
-    margin-bottom: 2rem;
+.page#unlock .blurred-identicons {
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    z-index: 0;
 }
 
+.page#unlock .blurred-identicons img {
+    width: 39.328rem;
+    opacity: 0.2;
+    -webkit-filter: blur(20px);
+    -moz-filter: blur(20px);
+    -o-filter: blur(20px);
+    -ms-filter: blur(20px);
+    filter: blur(20px);
+}
+
+.page#unlock .page-header,
+.page#unlock .page-body,
+.page#unlock .page-footer {
```

Comment:
I don't think that `.page-body` and `.page-footer` have the appropriate `position` for `z-index` to have an effect.

Reply:
I tested leaving them out, but these rules are necessary. Otherwise the background blurred identicons are partly in the foreground.